app/views/users/registration.html.erb

```diff
@@ -1,95 +1,72 @@
-
-<section class="section">
-<div class="registration-steps">
-  <div class="step one">
-    <span class="pink circle">1</span>
-    <p>Register with GitHub</p>
-  </div>
-  <div class="pink-line"></div> <%# css line %>
-  <div class="step two">
-    <span class="yellow circle">2</span>
-    <p>About You</p>
-  </div>
-   <div class="right-bar"></div> <%# css line %>
-   <div class="step three">
-    <span class="pink-open circle">3</span>
-    <p>Start Contributing</p>
-  </div>
-</div>
-</section>
 <section class="section reg-content">
-  <div>
-    <h1 class="title is-1">About you</h1>
-    <p>
-      Please select which email address you'd like
-      to use for Hacktoberfest, and read the
-      <a target="_blank" href="/details#participation-rules">Hacktoberfest
-      rules and values</a>.
-    </p>
-    <%= form_for :user, url: { action: 'register' }, method: :patch do |f| %>
+  <div class="registration-wrapper">
 
-      <div class="field">
-        <label class="label">Preferred email</label>
-        <div class="control">
-          <div class="select">
-            <%= f.select :email, @emails %>
-          </div>
-        </div>
-      </div>
+    <div class="registration-left">
+      <div class="register-header">Register</div>
+      <h1 class="title is-1">About you</h1>
+      <p>
+        Please select which email address you’d like to use for Hacktoberfest and read the <a target="_blank" href="/details#participation-rules">rules and values</a>.
+      </p>
+      <%= form_for :user, url: { action: 'register' }, method: :patch do |f| %>
 
-      <div class="field">
-        <label class="label">Role</label>
-        <div class="control">
-          <div class="select">
-            <%= f.select :category, @categories %>
+        <div class="field">
+          <div class="control">
+            <div class="select">
+              <%= f.select :email, @emails, :prompt => "Select prefered email address" %>
```

Comment:
Need to bring user categories back in here so users can select roles - that was one of those new requests from this year (got brought in on #482)